frontend/src/components/navbar/NavbarLinks/index.tsx

```diff
@@ -1,65 +1,27 @@
-import { Box } from "@material-ui/core";
-import Link from "next/link";
-import { useRouter } from "next/router";
 import React from "react";
 import links from "./links";
 import PermissionRequired from "@components/permissions/PermissionRequired";
-import { useStyles } from "./styles";
+import NavbarItem from "./NavbarItem";
 
 type Props = {
   loggedIn: boolean;
 };
 
 const NavbarLinks: React.VFC<Props> = ({ loggedIn }) => {
-  const classes = useStyles();
-  const router = useRouter();
-
   return (
     <>
-      {links.map((item) => (
-        <div key={item.id} className={classes.nav}>
-          <Link href={item.href}>
-            <a
-              className={[item.href == "/" + router.pathname.split("/")[1] ? "active" : "", classes.navItem].join(" ")}
-            >
-              {item.title}
-            </a>
-          </Link>
-          {item.dropdown && (
-            <div className={classes.dropdown}>
-              {item.dropdown.map((dropItem) => (
-                <Link key={dropItem.title} href={dropItem.href}>
-                  <a className={[router.pathname == dropItem.href ? "active" : "", classes.navItem].join(" ")}>
-                    {dropItem.title}
-                  </a>
-                </Link>
-              ))}
-            </div>
-          )}
-        </div>
-      ))}
-      {loggedIn ? (
-        <PermissionRequired permission="archive.view_archivedocument">
-          <Box position="relative">
-            <Link href="/archive">
-              <a className={[router.pathname == "/archive" ? "active" : "", classes.navItem].join(" ")}>Arkiv</a>
-            </Link>
-          </Box>
-        </PermissionRequired>
-      ) : (
-        <Box position="relative">
-          <a
-            className={classes.navItem}
-            href="https://www.ntnu.no/studier/mtiot"
-            target="_blank"
-            rel="noreferrer noopener"
-          >
-            Om studiet
-          </a>
-        </Box>
+      {links.map(
+        (link) =>
+          (!link.loginRequired || (link.loginRequired && loggedIn)) &&
+          (link.permissionRequired ? (
+            <PermissionRequired permission={link.permissionRequired}>
+              <NavbarItem link={link} />
+            </PermissionRequired>
+          ) : (
+            <NavbarItem link={link} />
+          ))
```

Comment:
This should function the same. If logged in, show regardless of loginRequired. If not, only show those without loginRequired.
```suggestion
          (loggedIn || !link.loginRequired) &&
          (link.permissionRequired ? (
            <PermissionRequired permission={link.permissionRequired}>
              <NavbarItem link={link} />
            </PermissionRequired>
          ) : (
            <NavbarItem link={link} />
          ))
```

Reply:
Good catch! Fixed in 0ca67f1.